### PR TITLE
Moving all the plugins into the API

### PR DESCRIPTION
**Pull request by stjohnjohnson**
Moving:
 - https://github.com/screwdriver-cd/plugin-jobs
 - https://github.com/screwdriver-cd/plugin-pipelines
 - https://github.com/screwdriver-cd/plugin-platforms
 - https://github.com/screwdriver-cd/plugin-builds
 - https://github.com/screwdriver-cd/plugin-login
 - https://github.com/screwdriver-cd/plugin-users

Resolves https://github.com/screwdriver-cd/api/issues/28

### CODE REVIEW

**Comment by tkyi** [comment]
👍  LGTM
giphy-11

File: plugins/builds/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Builds Plugin
```
Comment (d2lam) [inline plugins/builds/README.md]: Do you think it would be helpful to include a section in the main README that points to the README for these plugins?

**Comment by d2lam** [comment]
Small comments regarding the README. The rest LGTM 👍

**Comment by d2lam** [comment]
Sorry clicked the wrong button. Accidentally closed. Oops

File: plugins/builds/README.md (hunk 1 above)
Comment (stjohnjohnson) [inline plugins/builds/README.md]: Done!

**Comment by d2lam** [comment]
Looks good

**Pull request merged**